spec/services/find_third_party_project_activities_spec.rb

```diff
@@ -8,7 +8,24 @@
   let!(:organisation_project) { create(:third_party_project_activity, organisation: other_organisation) }
   let!(:other_project) { create(:third_party_project_activity) }
 
+  before(:all) do
+    Bullet.add_whitelist type: :unused_eager_loading, class_name: "Activity", association: :parent
+  end
+
+  after(:all) do
```

Comment:
We target the specific whitelist we added once we're done with all tests in the file, and remove it.